Kudu.Core/Helpers/PostDeploymentHelper.cs

```diff
@@ -275,11 +275,26 @@ private static void VerifyEnvironments()
                 var json = (Dictionary<string, object>)serializer.DeserializeObject(File.ReadAllText(functionJson));
 
                 object value;
-                var disabled = json.TryGetValue("disabled", out value) && (bool)value;
-                if (disabled)
+                // https://github.com/Azure/azure-webjobs-sdk-script/blob/a9bafba78a3a8092bfd61a8c7093200dae867efb/src/WebJobs.Script/Host/ScriptHost.cs#L1476-L1498
+                if (json.TryGetValue("disabled", out value))
                 {
-                    Trace(TraceEventType.Verbose, "Function {0} is disabled", functionName);
-                    return Enumerable.Empty<Dictionary<string, object>>();
+                    string stringValue = value.ToString();
+                    bool disabled;
+                    // if "disabled" is not a boolean, we try to expend it as an environment variable
+                    if (!Boolean.TryParse(stringValue, out disabled))
+                    {
+                        string expandValue = System.Environment.GetEnvironmentVariable(stringValue);
+                        // null/"" -> false, "1"/"true" -> true
+                        disabled = string.IsNullOrEmpty(expandValue) ? false :
+                            string.Equals(expandValue, "1", StringComparison.OrdinalIgnoreCase) ? true :
+                            string.Equals(expandValue, "true", StringComparison.OrdinalIgnoreCase);
```

Comment:
A bit simpler to just write:

    disabled = !string.Equals(expandValue, "1", StringComparison.OrdinalIgnoreCase) && !string.Equals(expandValue, "true", StringComparison.OrdinalIgnoreCase)